Bots/DotNet/Consumers/Composer/ComposerHostBotDotNet/.gitignore

```diff
@@ -0,0 +1,2 @@
+# files generated during the lubuild process
+generated/
```

Comment:
Use the git ignore from [Bots\DotNet\Consumers\Composer\SimpleHostBotComposer\.gitignore](https://github.com/southworks/BotFramework-FunctionalTests/blob/d731d0767107c66e250a1974144bfe4ce988127e/Bots/DotNet/Consumers/Composer/SimpleHostBotComposer/.gitignore).
Before pushing this change, make sure to start the bot at least once, it should have generated some files under the generated folder like in [Bots\DotNet\Consumers\Composer\SimpleHostBotComposer\generated](https://github.com/southworks/BotFramework-FunctionalTests/tree/d731d0767107c66e250a1974144bfe4ce988127e/Bots/DotNet/Consumers/Composer/SimpleHostBotComposer/generated).